docs/src/main/tut/typeclasses/monoid.md

```diff
@@ -7,70 +7,138 @@ scaladoc: "#cats.kernel.Monoid"
 ---
 # Monoid
 
-`Monoid` extends the [`Semigroup`](semigroup.html) type class, adding an 
-`empty` method to semigroup's `combine`. The `empty` method must return a 
-value that when combined with any other instance of that type returns the 
-other instance, i.e.
+`Monoid` extends the power of `Semigroup` by providing an additional `empty` value.
 
-```scala
-(combine(x, empty) == combine(empty, x) == x)
-```
-    
-For example, if we have a `Monoid[String]` with `combine` defined as string 
-concatenation, then `empty = ""`.
+```tut:book:silent
+trait Semigroup[A] {
+  def combine(x: A, y: A): A
+}
 
-Having an `empty` defined allows us to combine all the elements of some 
-potentially empty collection of `T` for which a `Monoid[T]` is defined and 
-return a `T`, rather than an `Option[T]` as we have a sensible default to 
-fall back to.
+trait Monoid[A] extends Semigroup[A] {
+  def empty: A
+}
+```
 
-First some imports.
+This `empty` value should be an identity for the `combine` operation, which means the following equalities hold
+for any choice of `x`.
 
-```tut:silent
-import cats._
-import cats.implicits._
+```
+combine(x, empty) = combine(empty, x) = x
 ```
 
-Examples.
+Many types that form a `Semigroup` also form a `Monoid`, such as `Int`s (with `0`) and `Strings` (with `""`).
 
-```tut:book
-Monoid[String].empty
-Monoid[String].combineAll(List("a", "b", "c"))
-Monoid[String].combineAll(List())
+```tut:reset:book:silent
+import cats.Monoid
+
+implicit val intAdditionMonoid: Monoid[Int] = new Monoid[Int] {
+  def empty: Int = 0
+  def combine(x: Int, y: Int): Int = x + y
+}
+
+val x = 1
 ```
 
-The advantage of using these type class provided methods, rather than the 
-specific ones for each type, is that we can compose monoids to allow us to 
-operate on more complex types, e.g.
- 
 ```tut:book
-Monoid[Map[String,Int]].combineAll(List(Map("a" -> 1, "b" -> 2), Map("a" -> 3)))
-Monoid[Map[String,Int]].combineAll(List())
+Monoid[Int].combine(x, Monoid[Int].empty)
+
+Monoid[Int].combine(Monoid[Int].empty, x)
+```
+
+# Exploiting laws: associativity and identity
```

Comment:
Ah I had a different intent for this section initially but went with a different direction and forgot to change it, oops :-)